app/models/katello/content_view.rb

```diff
@@ -604,6 +604,7 @@ def check_ready_to_publish!(importing: false, syncable: false)
         fail _("Import-only content views can not be published directly") if import_only? && !syncable
         check_composite_action_allowed!(organization.library)
         check_docker_repository_names!([organization.library])
+        check_orphaned_content_facets(organization.library)
```

Comment:
Can we run this for all environments instead of just library ?
```ruby
        organization.kt_environments.each do |env|
             check_orphaned_content_facets(env)
        end
```

Reply:
Updated to use cv.environments.